Makefile

```diff
@@ -6,7 +6,7 @@ ECARGS    ?=
 ECTOUT    ?= 10
 ECJOBS    ?= 0
 ECEXTRA   ?= --report=report.log
-ECPROVERS ?= Alt-Ergo Z3 CVC4
+ECPROVERS ?= Alt-Ergo@2.4 Z3@4.8 CVC4@1.8
```

Comment:
Given Why3 version bump, these should be bumped up as well?

Reply:
They are supported by Why 1.7.

The sole purpose of this PR is to allow versioning the provers so that EasyCrypt can compile with the new docker images that includes multiple version of the same provers.

Bumping them leads to `smt` errors. I already have a fix for them, but will do a new PR.